pkg/database/realm.go

```diff
@@ -288,12 +296,24 @@ func NewRealmWithDefaults(name string) *Realm {
 		LongCodeLength:      16,
 		LongCodeDuration:    FromDuration(24 * time.Hour),
 		SMSTextTemplate:     DefaultSMSTextTemplate,
-		AllowedTestTypes:    14,
+		AllowedTestTypes:    14, // This doesn't include user initiated report by default.
```

Comment:
Nit: we should probably AND these together so it's clear what "14" actually means